cmd/bootstrap/cmd/constraints.go

```diff
@@ -40,7 +40,8 @@ func checkConstraints(partnerNodes, internalNodes []model.NodeInfo) {
 	}
 
 	// check collection committee Byzantine threshold for each cluster
-	_, clusters := constructClusterAssignment(partnerNodes, internalNodes)
+	// for checking Byzantine constraints, the seed doesn't matter
```

Comment:
why doesn't matter?

Wouldn't different seed change the cluster assignment? I'm a bit concerned about having a check based on a wrong assignment.

How hard is it to get the real seed?

Reply:
The seed is used to shuffle the internal and partner nodes separately, then they are round-robinned to clusters: https://github.com/onflow/flow-go/pull/637/files#diff-07f8689affbbe785aa8d2d5e6a15d730b66f9f9af4b2c580db279e35345f2123R22-R23.

Because we use round-robin, the Byzantine constraints depend on the size of the `internals` and `partners` lists, not their order.